### PR TITLE
Connect to SSID prefix functions.

### PR DESCRIPTION
**Pull request by jkaravakis**
iOS 13+ users can use connectToProtectedSSIDPrefix and connectToSSIDPrefix when they want to match a known SSID prefix, but don’t have a full SSID. If the device finds multiple Wi-Fi networks whose SSID string matches the given prefix, it selects the network with the greatest signal strength.

Added connectToProtectedSSIDPrefix and connectToSSIDPrefix to README, and fixed "isWeb" to "isWep" in documentation with detailed explanation.

### CODE REVIEW

**Comment by JuanSeBestia** [comment]
Now I have no means of how to test it, but it looks good.

I will test it on Tuesday

File: ios/RNWifi.m [hunk 1]
```diff
@@ -53,51 +53,74 @@ + (BOOL)requiresMainQueueSetup
 RCT_EXPORT_METHOD(connectToSSID:(NSString*)ssid
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject) {
-    
+
     if (@available(iOS 11.0, *)) {
         NEHotspotConfiguration* configuration = [[NEHotspotConfiguration alloc] initWithSSID:ssid];
-        configuration.joinOnce = true;
-        
+        configuration.joinOnce = false;
```
Comment (JuanSeBestia) [inline ios/RNWifi.m]: Maybe some users want to use in true, what do you think to turn this to param?

Comment (jkaravakis) [reply ios/RNWifi.m]: I have made that change as a remedy to issue #2 which I was experiencing as well.

Comment (JuanSeBestia) [reply ios/RNWifi.m]: let me observe how it behaves in Android, to make this decision

Comment (jkaravakis) [reply ios/RNWifi.m]: Hello @JuanSeBestia, any feedback on this PR?

Comment (shamilovtim) [reply ios/RNWifi.m]: @jkaravakis imo these changes are unrelated to the PR and don't belong in it. why not just set them back so the main functionality of the PR can get merged?

File: ios/RNWifi.m [hunk 2]
```diff
@@ -53,51 +53,74 @@ + (BOOL)requiresMainQueueSetup
...
 RCT_EXPORT_METHOD(connectToProtectedSSID:(NSString*)ssid
                   withPassphrase:(NSString*)passphrase
                   isWEP:(BOOL)isWEP
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject) {
-    
+
     if (@available(iOS 11.0, *)) {
         NEHotspotConfiguration* configuration = [[NEHotspotConfiguration alloc] initWithSSID:ssid passphrase:passphrase isWEP:isWEP];
-        configuration.joinOnce = true;
```
Comment (shamilovtim) [inline ios/RNWifi.m]: imo this belongs in a separate PR so this can get merged as is and not effect other functionality

File: ios/RNWifi.m (hunk 1 above)
Comment (jkaravakis) [reply ios/RNWifi.m]: I agree, will revert back to `joinOnce = true`.

File: ios/RNWifi.m [hunk 3]
```diff
@@ -53,51 +53,74 @@ + (BOOL)requiresMainQueueSetup
...
                 reject(@"nehotspot_error", @"Error while configuring WiFi", error);
             } else {
                 resolve(nil);
             }
         }];
-        
+
     } else {
-        reject(@"ios_error", @"Not supported in iOS<11.0", nil);
+        reject(@"ios_error", @"Not supported in iOS<13.0", nil);
```
Comment (JuanSeBestia) [inline ios/RNWifi.m]: Nope, in 12 and 11 work with worth combination

Comment (jkaravakis) [reply ios/RNWifi.m]: @JuanSeBestia good catch, I must have accidentally altered that error message for conectToProtectedSSID.

Comment (jkaravakis) [reply ios/RNWifi.m]: @JuanSeBestia I fixed it with the latest commit.

**Comment by jkaravakis** [comment]
I have added @shamilovtim's suggested connectToSSIDPrefix function to the scope of this PR. I have kept joinOnce as true as this is being addressed in #33.
